packages/react/src/themes/teams/siteVariables.ts

```diff
@@ -61,6 +61,17 @@ export const lineHeightMedium = 1.4286
 export const lineHeightLarge = 1.3333
 export const lineHeightLarger = 1.3333
 
+//
+// Z-INDEX
+//
+export const zIndexes = {
+  background: 0,
+  foreground: 1,
+  menuItem: 2,
+  overlay: 1000,
+  debug: 999999999,
```

Comment:
I like the system but maybe it should be more flexible - it's possible that some component might need a value between 1 & 2.  Can we use something like:
10
50
1000

Reply:
Sure, we can introduce new values whenever we have a need, i.e. for a new component
Current list of values was just extracted from existing styles

Reply:
Let's not change the actual values because we may introduce breaking changes for the theme. After we make sure that the necessary updates are done on client's side, we can play and change the values.

Reply:
Yep, that's my plan as well :)